### PR TITLE
[RFC] Remove admin alerts, there are better mechanisms for this

### PR DESCRIPTION
**Pull request by wenottingham**

### CODE REVIEW

**Comment by ryanpetrello** [comment]
Do email-based Notification Templates use any of this under the hood?  

I expect they might, given that this:

https://github.com/ansible/awx/pull/5182/files#diff-0091f8a83b63dafea8313c794ba726b3L400

...matches this:

https://github.com/ansible/awx/blob/devel/awx/main/notifications/email_backend.py#L4

I'm all for removing the periodic monitoring task that emails the admins, but the `awx.settings` changes make me a bit nervous (I expect Django probably just overrides them with similar defaults anyways if you remove what we've set).

**Comment by wenottingham** [comment]
https://github.com/ansible/awx/blob/devel/awx/main/notifications/email_backend.py#L14 it doesn't look like it?

**Comment by wenottingham** [comment]
More detailed look at the settings in those files:

All of the fields except for EMAIL_BACKEND are fields on the notification template themselves, so I presume it works without them in settings. The only settings in the awx/settings/defaults.py that are not the django defaults are DEFAULT_FROM_EMAIL and EMAIL_SUBJECT_PREFIX; both of these are overriden by the notification backend.

Will run some tests regardless.

**Comment by matburt** [comment]
None of these are shared with the notification system. These were originally intended to be used to alert users when the llicense was about to expire (for example).

Given that we are moving away from this kind of thing. I vote for removal. The original requirement for this came from a certain individual named @wenottingham :D

**Review by matburt — APPROVED** [review]

File: awx/main/conf.py [hunk 1]
```diff
@@ -54,15 +54,6 @@
...
 )
 
-register(
-    'TOWER_ADMIN_ALERTS',
-    field_class=fields.BooleanField,
-    label=_('Enable Administrator Alerts'),
-    help_text=_('Email Admin users for system events that may require attention.'),
-    category=_('System'),
-    category_slug='system',
-)
```
Comment (AlanCoding) [inline awx/main/conf.py:64]: Have we ever tested removing a setting? This is where I get worried about how the UI would PATCH with the default values, so it's almost certain the pre-migration this value will be set for anyone who ever saved settings. Should be simple to test.

**Review by ryanpetrello — APPROVED** [review]

File: awx/main/conf.py (hunk 1 above)
Comment (wenottingham) [reply awx/main/conf.py:64]: https://github.com/ansible/awx/commit/af84b25726326842d3e6a4b9f9de5e670945d070 is an example; we didn't do anything with migrations there.

Comment (ryanpetrello) [reply awx/main/conf.py:64]: Yep, removing settings is fine as long as you don't reference them in code somewhere.  Having stray records in the `conf_setting` table shouldn't hurt anything.

**Pull request merged**